app/Console/Commands/VariantFilesize.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\SizeVariant;
+use Illuminate\Console\Command;
+
+class VariantFilesize extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'lychee:variant_filesize {limit=50 : number of photos to process at once (0 means all)}';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Set filesize of size variants if missing';
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return mixed
+	 */
+	public function handle()
```

Comment:
The return values are inconsistent. You return `false`, if the command does not find anything to do, and you return nothing (or `void`) in case of success.

Actually, the return type should be `int` and the method should return `0` on success and a non-zero POSIX error code in case of failure. For the sake of simplicity return `-1` in case of doubt.

```suggestion
	 * @return int
	 */
	public function handle(): int
```

Reply:
To be honest I took inspiration from `ExifLens` command as I didn't know the "Laravel-way" to do stuff. Didn't know that PHP had type hints either. Happy to learn!

See c0c52ae6e44bd36f4f0419945b2713ed367bf48f.

Reply:
Most of the console commands are really ugly, better don't take them for inspiration.

May I ask why you introduced a separate variable `exit_code` if you constantly return `0` in all cases? Besides this minor negligibility, it looks good. I don't mind if the variable remains there, but personally it strikes me as unnecessary.

IMHO, this issue is resolved.

Reply:
`exit_code` is `-1` if any update fails (_e.g._ [there](https://github.com/LycheeOrg/Lychee/commit/c0c52ae6e44bd36f4f0419945b2713ed367bf48f#diff-f9ded72faf81271d63a161dffa3455545a42249acf10efcd0bab9b5ceb483b7eR55)). 

To do that I needed to "bind" `exit_code` to the closure; this is because I don't want to exit on failure but try to do the maximum.

Is it ok ?

Reply:
Sure it is. My fault that I did not spot that.